module/item/item-sheet.js

```diff
@@ -319,6 +365,23 @@ export async function useItem(event){
         const item = actor.items.get(itemId);
         await item.update({"system.quantity": Number(duplicate(item.system.quantity)) - 1});
     }
+
+	// if the item is a food buff
+	if (game.ptu.data.items.find(i => i.name.toLowerCase().includes(itemName.toLowerCase())).system.consumable == 2){
+		//if there is a target actor add the food buff to the target actor, else add foodbuff to the user
+		
+		const effectedActor = targetedActor ? targetedActor : actor;
+		//if the target actor already has a food buff, add the new food buff to the list
+		if (effectedActor.system.digestionBuff.trim() != "" && effectedActor.system.digestionBuff.trim().toLowerCase() != "none"){
+			const buffArray = effectedActor.system.digestionBuff.split(", ");
+			buffArray.push(itemName);
+			const newBuffString = buffArray.join(", ");
+			await effectedActor.update({"data.digestionBuff": newBuffString});
+		}
+		else{
+			await effectedActor.update({"data.digestionBuff": itemName});
+		}		
```

Comment:
```suggestion
	const buffArray = effectedActor.system.digestionBuff.split(", ").filter(b => b.trim() != "");
	buffArray.push(itemName);
	const newBuffString = buffArray.join(', ');
	await effectedActor.update({"data.digestionBuff": newBuffString});
```
Can be simplified, but up to preference tbh. This way no if check is needed.

Reply:
Also may be smarter to check `string.trim().length != 0` instead of `!= ""`